src/Core/SettingsChangesHistory.cpp

```diff
@@ -57,6 +57,12 @@ String ClickHouseVersion::toString() const
 /// Note: please check if the key already exists to prevent duplicate entries.
 static std::initializer_list<std::pair<ClickHouseVersion, SettingsChangesHistory::SettingsChanges>> settings_changes_history_initializer =
 {
+    {"24.13",
+        {
+            {"output_format_always_quote_identifiers", false, false, "New setting."},
+            {"output_format_identifier_quoting_style", "Backticks", "Backticks", "New setting."}
+        }
+    },
```

Comment:
The version number is actually YY.MM, so 24.13 is not a valid version. These changes should go to 24.9